language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -261,8 +261,8 @@ genType curModName = go
                     ( con' <> "<" <> T.intercalate ", " ts' <> ">"
                     , ser <> "(" <> T.intercalate ", " sers <> ")"
                     )
-        TSyn _ -> error "TODO: genType, type synonym"
         TCon _ -> error "IMPOSSIBLE: lonely type constructor"
+        TSynApp{} -> error "TODO: genType, type synonym"
```

Comment:
This should be an impossible not a TODO since type synonyms are not serializable.

Reply:
done